### DISCUSSION TITLE
没法导入swagger格式的文件

### DISCUSSION THREAD

**Discussion by kooera** [Postcatlab/postcat#134] [Extension Feedback]
无法导入swagger格式文件有点遗憾，我又用回apifox了，尴尬了，要是支持swagger一顶到底

**Comment by scarqin** [comment]
@kooera 支持的，通过插件。
<img width="1182" alt="image" src="https://user-images.githubusercontent.com/15225835/188086402-40d4e338-c294-4f68-bc6a-e12d3cfe3e1e.png">
![image](https://user-images.githubusercontent.com/15225835/188086592-79fd9761-537a-4470-8617-2c94d0525b23.png)
![image](https://user-images.githubusercontent.com/15225835/188086570-bdd22c9c-4f91-4e21-b004-faa7d5d09295.png)
别走，快来一顶到底 :)

**Reply by kooera** [reply]
试过了，.net swagger v3 的导不进去，没有提示，也没有成功

**Reply by scarqin** [reply]
@kooera 什么版本，内容方便发我邮箱吗 scarqin@gmail.com？

**Reply by kooera** [reply]
![image](https://user-images.githubusercontent.com/18741234/189799445-b7c5e710-f74c-421a-982f-c017088ccc6a.png)  
私发了，有时间帮忙看一下，虽然都遵循openapi的规则，应该是有一些多余的属性吧

**Reply by scarqin** [reply]
@kooera 收

**Reply by buqiyuan** [reply]
@kooera Hello~ 感谢反馈！已修复，更新`导入OpenAPI插件`到 `1.0.16` 再试下导入，有问题欢迎随时反馈~

**Comment by kooera** [comment]
谢谢！您的邮件可能我现在没有时间回复您！我会尽快回复的
